.github/workflows/ci_frontend.yml

```diff
@@ -1,71 +1,71 @@
-# limitations under the License.
+#jobs:
+#  Compile-check:
+#    runs-on: ${{ matrix.os }}
+#    strategy:
+#      matrix:
+#        os: [ubuntu-latest, macos-latest]
+#    steps:
+#      - uses: actions/checkout@v2
+#      # In the checkout@v2, it doesn't support git submodule. Execute the commands manually.
+#      - name: checkout submodules
+#        shell: bash
+#        run: |
+#          git submodule sync --recursive
+#          git -c protocol.version=2 submodule update --init --force --recursive --depth=1
+#      - name: Set up Node.js
+#        uses: actions/setup-node@v1
+#        with:
+#          version: 8
+#      - name: Compile
+#        run: |
+#          cd dolphinscheduler-ui
+#          npm install node-sass --unsafe-perm
+#          npm install
+#          npm run lint
+#          npm run build
 #
-
-name: Frontend
-
-on:
-  push:
-    paths:
-      - '.github/workflows/ci_frontend.yml'
-      - 'dolphinscheduler-ui/**'
-  pull_request:
-    paths:
-      - '.github/workflows/ci_frontend.yml'
-      - 'dolphinscheduler-ui/**'
-
-jobs:
-  Compile-check:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest, macos-latest]
-    steps:
-      - uses: actions/checkout@v2
-      # In the checkout@v2, it doesn't support git submodule. Execute the commands manually.
-      - name: checkout submodules
-        shell: bash
-        run: |
-          git submodule sync --recursive
-          git -c protocol.version=2 submodule update --init --force --recursive --depth=1
-      - name: Set up Node.js
-        uses: actions/setup-node@v1
-        with:
-          version: 8
-      - name: Compile
-        run: |
-          cd dolphinscheduler-ui
-          npm install node-sass --unsafe-perm
-          npm install
-          npm run lint
-          npm run build
-
-  License-check:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      # In the checkout@v2, it doesn't support git submodule. Execute the commands manually.
-      - name: checkout submodules
-        shell: bash
-        run: |
-          git submodule sync --recursive
-          git -c protocol.version=2 submodule update --init --force --recursive --depth=1
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
-        with:
-          java-version: 1.8
-      - name: Check
-        run: mvn -B apache-rat:check
+#  License-check:
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v2
+#      # In the checkout@v2, it doesn't support git submodule. Execute the commands manually.
+#      - name: checkout submodules
+#        shell: bash
+#        run: |
+#          git submodule sync --recursive
+#          git -c protocol.version=2 submodule update --init --force --recursive --depth=1
+#      - name: Set up JDK 1.8
+#        uses: actions/setup-java@v1
+#        with:
+#          java-version: 1.8
+#      - name: Check
+#        run: mvn -B apache-rat:check
```

Comment:
During the project incubation stage, it is not recommended to remove the license check

Reply:
I just want to test ut ci, so I removed the other ci for more quickly result, I will add it back when I finished the test.